content/en/metrics/guide/agent-filtering-for-dogstatsd-custom-metrics.md

```diff
@@ -0,0 +1,126 @@
+
+---
+title: Agent-Side Filtering for DogStatsD Custom Metrics 
+private: true
+further_reading:
+- link: "/metrics/custom_metrics/"
+  tag: "Documentation"
+  text: "Learn more about Custom Metrics"
+- link: "/account_management/billing/custom_metrics/?tab=countrate"
+  tag: "Documentation"
+  text: "Custom Metrics Billing"
+- link: "/metrics/metrics-without-limits/"
+  tag: "Documentation"
+  text: "Metrics without Limits™"
+- link: "/metrics/volume/"
+  tag: "Documentation"
+  text: "Metrics Volume Management"
+- link: "https://www.datadoghq.com/blog/custom-metrics-governance/"
+  tag: "Blog"
+  text: "Best practices for end-to-end custom metrics governance"
+---
+
+{{< callout url="https://www.datadoghq.com/" btn_hidden="true" header="false" >}}
+Agent-side filtering for DogStatsD custom metrics is in Preview.
+{{< /callout >}} 
+
+## Overview
+
+Agent-side filtering enables you to block unused or unwanted DogStatsD custom metrics directly at the Datadog Agent, before sending them to Datadog. This can significantly reduce both indexed and ingested custom metric volume.
```

Comment:
```suggestion
Agent-side filtering enables you to filter out unused or unwanted DogStatsD custom metrics directly at the Datadog Agent, before sending them to Datadog. This can significantly reduce both indexed and ingested custom metric volume.
```